client/src/app/[tenant]/agents/[taskId]/[taskSchemaId]/playground/playgroundModelOutputContent.tsx

```diff
@@ -82,6 +83,7 @@ export function PlaygroundModelOutputContent(props: ModelOutputContentProps) {
     isHideModelColumnAvaible,
     hideModelColumn,
     isProxy,
+    hasProxyInput,
```

Comment:
You should move to a context to avoid having to pass throughout the component hierarchy